_includes/releases/v23.1/v23.1.0-alpha.5.md

```diff
@@ -0,0 +1,166 @@
+- Using subqueries in UDFs without an `AS` clause is now supported, for consistency with the syntax supported outside of UDFs. [#97515][#97515]
+- Fixed a rare bug existing since before v22.1.x that could cause a projected expression to replace column references with the wrong values. [#97554][#97554]
+- Cross-descriptor validation on lease renewal is now disabled, because it can starve schema changes when there are many descriptors with many foreign key references [#97630][#97630]
+- Fixed a bug with pagination on the Schema Insights page. [#97640][#97640]
+- Columns referenced in partial index predicates and partial unique constraint predicates can no longer be dropped. The `ALTER TABLE .. DROP COLUMN` statement now returns an error with a suggestion to drop the indexes and constraints first. This is a temporary safe-guard to prevent users from hitting [#96924][#96924]. This restriction will be lifted when that bug is fixed. [#97372][#97372]
+- Jobs page properly shows error state when we receive an error during data fetching. [#97486][#97486]
+- Fixed a bug introduced in v22.1 that caused the internal error "no bytes in account to release ...". [#97750][#97750]
+- The `COPY FROM` command now respects the `statement_timeout` and `transaction_timeout` cluster settings. [#97808][#97808]
+- `COPY FROM` commands now appears in the output of the `SHOW QUERIES` command. [#97808][#97808]
+- Fixed an error where querying a `pg_catalog` table included information about a temporary table created in another session. [#97727][#97727]
+
+<h3 id="v23-1-0-alpha-5-performance-improvements">Performance improvements</h3>
+
+- If the session setting `optimizer_use_improved_split_disjunction_for_joins` is `true`, the optimizer now creates a better query plan in some cases where an inner, semi, or anti join contains a join predicate with a disjuction (`OR` condition). [#97696][#97696]
+
+<h3 id="v23-1-0-alpha-5-miscellaneous">Miscellaneous</h3>
+
+- UDFs can now return the `RECORD` result type, which represents any tuple. For example, `CREATE FUNCTION f() RETURNS RECORD AS 'SELECT * FROM t' LANGUAGE SQL;` is equivalent to `CREATE FUNCTION f() RETURNS t AS 'SELECT * FROM t' LANGUAGE SQL;`. [#97199][#97199]
+- The parameters for delegated snapshots have been marked as non-public. [#97408][#97408]
+- Fixed an error when calling `CREATE OR REPLACE FUNCTION` with a user-defined return type if the UDT was modified after the original UDF was created. The command now succeeds as long as the function body returns output that matches the modified UDT. [#97274][#97274]
+
+<h4 id="v23-1-0-alpha-5-changes-without-release-note-annotation">Changes without release note annotation</h4>
+
+- Columns with referenced constraints can now be dropped. [#97579][#97579]
+- Index cascades with a dependent inbound foreign key can now be dropped. [#97065][#97065]
+
+<h3 id="v23-1-0-alpha-5-doc-updates">Doc updates</h3>
```

Comment:
Is this header still visible? If so, and there's no content, it should be removed. Ditto the comment below.